insightedge-packager/src/main/scala/org/insightedge/spark/packager/Launcher.scala

```diff
@@ -28,6 +28,20 @@ import org.insightedge.spark.packager.Utils._
   */
 object Launcher {
 
+  def getXapLocation(edition: String, projectBasedir: String): String = {
+    println(s"edition: " + edition)
+    val prefix = s"$projectBasedir/insightedge-packager/target/"
+    if (edition.equalsIgnoreCase("premium")) {
+      prefix + "xap-premium.zip"
+    }
+    else if(edition.equalsIgnoreCase("community")) {
```

Comment:
please follow standard scala/java our code conventions (`else` statement should be on the same line)